src/App.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './App.css';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import Sidebar from './components/sidebar/Sidebar';
+import MainPage from './components/main_page/MainPage';
+import LandingPage from './components/landing_page';
+import Login from './components/auth/login';
+import Signup from './components/auth/signup';
+import ReservationsPage from './components/reservations';
+import NewReservation from './components/newReservation';
+import CourseDetail from './components/CourseDetail';
+import CourseForm from './components/course/courseForm';
+import DeleteCoursePage from './components/DeleteCoursePage';
+import Logout from './components/logout';
+
+function App() {
```

Comment:
- [ ] When I try to create a user, this happens:

![image](https://github.com/ab-noori/EduPlanner-Frontend/assets/57421823/d7ac641a-5af0-4715-bd5e-27cfe84da899)

![image](https://github.com/ab-noori/EduPlanner-Frontend/assets/57421823/b4673c6e-8987-4e7a-9317-7b761991a8a4)


Kindly, make it so users can be created, and logged in. 👍 

_note: This might work for you because your session storage is already populated, but not for new users that clone your project.  I suggest deleting your session storage and/or trying this on an incognito tab in order to properly test the functionality.  If this is not the case, I would check the CORS configuration._

_note 2: This means I didn't have access to the core functionality.  This will be taken a look in the next review._